lib/services/ratings/app_ratings_parser.rb

```diff
@@ -0,0 +1,24 @@
+class AppRatingsParser
+  attr_reader :response
+  def initialize(response)
+    @response = response
+  end
+
+  def self.call(response)
+    new(response).call
+  end
+
+  def call
+    parse_ratings
+  end
+
+  private
+
+  def parse_ratings
+    if parsed_response == JSON.parse(response)
+      parsed_response['content']['ratings']
```

Comment:
думаю будет лучше что-то типо 
```suggestion
     JSON.parse(response).dig('content', 'ratings')
```
в одну строку